funcx_endpoint/funcx_endpoint/executors/high_throughput/funcx_worker.py

```diff
@@ -225,13 +225,27 @@ def cli_run():
         debug=args.debug,
     )
 
-    worker = FuncXWorker(
-        args.worker_id,
-        args.address,
-        int(args.port),
-        worker_type=args.type,
-    )
-    worker.start()
+    # Redirect the stdout and stderr
```

Comment:
This looks like some significant change to output log handling that isn't mentioned in the changelog or PR title. Can you elaborate at least in the changelog what a user should be expecting to be different here?

Reply:
I added mention of this change to the changelog and the description of this PR.